playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
I agree with using --skip-tags for this.

I don't like the idea of merging playbooks as I think it makes sense to have two separate host types. Also, if they're combined, how do I just create new minions? Or just new masters? We need to be able to scale these independent of each other.

@jwhonce What are your thoughts on this. Is it necessary for cluster.sh to do the configure twice for some reason (like once to initially configure and once to register)?